cmd/collector/collector.go

```diff
@@ -173,13 +181,41 @@ func run() error {
 		}
 	}()
 
+	var server *http.Server
+	// Start the HTTP server in a separate goroutine
+	go func() {
+		mux := http.NewServeMux()
+		mux.HandleFunc("/records", flowRecordHandler)
+		mux.HandleFunc("/reset", resetRecordHandler)
+
+		server = &http.Server{
+			Addr:              ":8080",
+			Handler:           mux,
+			ReadHeaderTimeout: 5 * time.Second,
+		}
+
+		klog.InfoS("Server listening on :8080")
+		if err := server.ListenAndServe(); err != http.ErrServerClosed {
+			klog.Fatal(err, "ListenAndServe error")
```

Comment:
`klog.Fatal` is not really mean to be used like this IIRC. This won't format very nicely. See https://go.dev/play/p/d7u96VgMq46

Reply:
Changed it to Fatalf instead. Thanks for the suggestion.